docker-compose.yml

```
@@ -0,0 +1,39 @@
+version: '3.9'
+
+services:
+  ln_giveaway_backend:
+    platform: linux/amd64
+    build: .
+    container_name: ln_giveaway_backend
+    command: ./entrypoint.sh
+    restart: on-failure
+    volumes:
+      - ./:/usr/src/app/
+    env_file:
+      - ./.env
+    depends_on:
+      - ln_giveaway_db
+    ports:
+      - 6000:6000
+    environment:
+      - CONSUMER_KEY=${CONSUMER_KEY}
+      - CONSUMER_SECRET=${CONSUMER_SECRET}
+      - ACCESS_KEY=${ACCESS_KEY}
+      - ACCESS_SECRET=${ACCESS_SECRET}
+      - LN_API_KEY=${LN_API_KEY}
+      - SQLALCHEMY_DATABASE_URI=postgresql://ln_giveaway:ln_giveaway@ln_giveaway_db:5432/ln_giveaway_dev
+      - SQLALCHEMY_TRACK_MODIFICATIONS=False
+
+  ln_giveaway_db:
+    image: postgres
```

we should specify the Postgres image version e.g.
```suggestion
    image: postgres:14.1
```